src/HTTPCommands.cc

```diff
@@ -277,11 +277,22 @@ size_t HTTPRequest::ReadCallback(char *buffer, size_t size, size_t n, void *v) {
 		return CURL_READFUNC_ABORT;
 	}
 
+	payload->m_parent.m_log.Log(
```

Comment:
Let's switch this log level to `Dump`.

When you are generating strings for a log line, wrap it in a conditional so it's only done if the log mask is enabled.